### DISCUSSION TITLE
wz_mini_hacks on a the Wyze Outdoor cam

### DISCUSSION THREAD

**Discussion by EG-John** [gtxaspec/wz_mini_hacks#489] [Q&A]
Hello, thank you all for all your hard work.. I have it working on my Wyze v3 cameras.  Can I load this on my Wyze Outdoor Cameras?

**Comment by bjs-pdx** [comment]
Highly unlikely.  The Outdoor camera runs on battery and would be out of power very quickly.  That's one of the reasons Wyze doesn't support continuous recording on them, nor have they EVER released an RTSP firmware for them.

**Comment by Bob-AOL** [comment]
If the camera is based on the T31 platform it may accept the wz_mini bootloader and load wz_mini from an SD card. Getting past the base station with wz_mini enabled RTSP may be another story. Easy enough to try.

**Comment by Viss** [comment]
I bought a 'wyze battery cam pro' hoping I could use this firmware with it - I also bought the accompanying solar panel. How can I tell if this camera is T31 based?

**Comment by gtxaspec** [comment]
the new battery cam pro uses a different platform, it's not ingenic based, so this mod won't work with it.

**Comment by Viss** [comment]
Crap. i guess i get to return the thing then. Oh well. was worth a shot.

**Comment by qacollective** [comment]
I too purchased Wyze Battery Cams hoping that I'd be able to use this repo with them, but now that I found this thread ... it looks like I'll be packaging them all back up to return to Amazon again.  I think the hardware is good but the app is just a terrible let down, unless you get cam plus I guess - which was half the idea of using this toolset - to use them offline.  

Anyway, perhaps it would be a good idea to explicitly mention _incompatibility_ in the wiki, might save people like me some time.
